modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -24,6 +24,16 @@ supported by {product-title}
 endif::openshift-rosa[]
 and which CSI features they support, such as volume snapshots and resize.
 
+ifndef::openshift-rosa[]
+[IMPORTANT]
+====
+If your CSI driver is not listed in the following table, you must follow the installation instructions provided by your CSI storage vendor to use their supported CSI features.
+====
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+In addition to the drivers listed in the following table, ROSA functions with CSI drivers from third-party storage vendors. Red Hat does not oversee third-party provisioners or the connected CSI drivers and the vendors fully control source code, deployment, operation, and Kubernetes compatibility. These volume provisioners are considered customer-managed and the respective vendors are responsible for providing support. See the link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.html#rosa-policy-responsibilities_rosa-policy-responsibility-matrix[Shared responsibilities for {product-title}] matrix for more information. 
```

Comment:
Not sure who asked for this (relevant though) but shouldn't it also apply to ARO?

Reply:
yeah agree, it worked fine on ARO, HCP Azure cluster as well. 
https://issues.redhat.com/browse/STOR-1719?focusedId=24599828&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24599828

Reply:
@bmcelvee I believe that this is your content. OK with adding ARO to ifdef?

Reply:
Thank you for bumping this @lpettyjo! It makes sense that it applies to ARO, but we don't have that content at this point in time. Looping in @bhardesty too.

Reply:
@lpettyjo Sure, go ahead and add it! We don't currently publish any docs for ARO, but that will likely change in the future for ARO HCP.

Reply:
Done.